### PR TITLE
Remove dependency on gadgets library

### PR DESCRIPTION
**Pull request by donnapep**

### CODE REVIEW

File: rise-rss.html [hunk 1]
```diff
@@ -357,13 +357,6 @@
        *
        */
       go: function() {
-        if (!offlineWindow) {
```
Comment (donnapep) [inline rise-rss.html]: @stulees Can you review this please? I believe that we got rid of offline player logic in the Storage component, and so I'm wondering if it's safe to rip out this entire check here. Thx.

**Comment by donnapep** [comment]
@stulees @rodrigopavezi Please review. @stulees Let me know if it's OK to get rid of [this section](https://github.com/Rise-Vision/rise-rss/pull/26/commits/bf4480fa1d409110f72a7e417ff213b4d95f19a7#diff-fe7ec79d64cacd6f8ea72fea48df599eL360) in its entirety. Thx.

File: circle.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 machine:
   node:
-    version: 6.6.0
+    version: stable
```
Comment (rodrigopavezi) [inline circle.yml:3]: @donnapep nice one!!

**Comment by stulees** [comment]
@donnapep 👍 The use of gadgets was for using `makeRequest` to get data, not needed anymore.

**Comment by donnapep** [comment]
@stulees Thanks. So the `!offlineWindow` check isn't needed either? If that's the case, should we think about getting rid of all of the offline Player code?

**Comment by stulees** [comment]
@donnapep Offline player code is/was separate to makeRequest. If the component is/was running in Offline Player, than requesting data was handled via `postMessage` and that logic and code still remains. I'm not sure what's happened with Offline Player, so I'm not certain it's okay to remove that code.

**Comment by donnapep** [comment]
@stulees OK, in that case I'm not sure we can get rid of the check for `gadgets` after all. If we eliminate it, then there's no possible way that [this line](https://github.com/Rise-Vision/rise-rss/blob/master/rise-rss.html#L371) would ever execute.

I think we need to find out if we need to worry about this offline player code, and rip it all out if we don't.

@rodrigopavezi FYI.

**Comment by stulees** [comment]
@donnapep I agree on finding out if we need to concern ourselves about offline player as it would be better to rip all of it out completely. 

However, not sure I follow you about that line in your last comment not executing if we remove the block of code in this PR. The value of `offlineWindow` will be `undefined`, therefore https://goo.gl/vGQZLV will execute. Or am I missing something?

**Comment by donnapep** [comment]
@tejohnso We've got a lot of code in this component that relies on offline player and communicating via `postMessage`. We're trying to figure out if this offline functionality is now obsolete and should be removed. Any thoughts on that? Thx.

**Comment by tejohnso** [comment]
@donnapep It's obsolete now but may end up going back in (in a different form) later on.

**Comment by donnapep** [comment]
@stulees Given what Tyler said about offline player, I think it's OK to remove that code and eliminate the rest of the offline player code at some later time. Do you agree?

**Comment by stulees** [comment]
@donnapep Sounds good

**Pull request merged**